src/components/editor/Wysiwyg/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { forwardRef, useImperativeHandle, useRef } from 'react';
+import { Editor } from '@toast-ui/react-editor';
+import '@toast-ui/editor/dist/toastui-editor.css';
+import './Wysiwyg.scss';
+import fontSize from 'tui-editor-plugin-font-size';
+import 'tui-editor-plugin-font-size/dist/tui-editor-plugin-font-size.css';
+
+export interface WysiwygRef {
+  addImg: () => void,
+  bold: () => void,
+  heading: () => void,
+  paragraph: () => void,
+  through: () => void,
+}
+
+const Wysiwyg = forwardRef((_props, ref) => {
+  const editorRef = useRef<Editor>(null);
+  useImperativeHandle(ref, () => ({
+    addImg() {
+      if (editorRef.current) {
+        editorRef.current.getInstance().exec('addImage', { imageUrl: 'https://picsum.photos/200/300' });
+      }
```

Comment:
사소하지만, optional chaining 으로 로직을 개선할 수 있어 보여요

```suggestion
      editorRef.current?.getInstance().exec('addImage', { imageUrl: 'https://picsum.photos/200/300' });
```

Reply:
아 그렇네요.. 수정하겠습니다. 감사합니다!